third_party/ijar/test/BUILD

```diff
@@ -1,3 +1,9 @@
+load("@rules_java//java:java_binary.bzl", "java_binary")
```

Comment:
ijar is actually checked in internally, I sent a CL to fix this. We can merge the other two changes via gerrit.

Reply:
Thanks, I removed ijar from the PR.